packages/material-ui-lab/src/DataGrid/DataGrid.d.ts

```diff
@@ -27,6 +27,12 @@ export interface DataProviderType {
   getList: (params: DataProviderGetListParams) => Promise<any[]>;
 }
 
+export type PagingOptions = {
```

Comment:
I think that flattening the props will help for the documentation's descriptions.

Reply:
So they should all the top level in your opinion?

Reply:
yes, until we look into code splitting.